src/arith_uint256.cpp

```diff
@@ -10,6 +10,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include "crypto/common.h"
```

Comment:
@Diapolo Every code cleanup -- including those authored by myself -- has review costs and potentially disrupts more important functional work that actually changes behavior for the users.

Style takes a back seat to more practical changes, as it must.  It sucks when it's your changes.

Include file ordering is just not important enough to us to care about maintaining it in our own changes.  We absolutely value your contributions, but you have to understand how to work with everyone else in the project and judge your impact on other developers, not just the codebase.

We don't have rules, we have guidelines.